api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -82,7 +78,7 @@ private void checkAndAddRegistered(WorkspaceIngest workspaceIngest) {
   @Override
   public boolean getFirecloudStatus() {
     try {
-      new StatusApi().status();
+      new FireCloudConfig().statusApi(configProvider.get()).status();
```

Comment:
We should instead inject Provider&lt;StatusApi&gt; into the constructor and use it here, I think.

Reply:
OK. Let me know if I can help.